packages/linear-genome-view/src/BasicTrack/blockBasedTrackModel.ts

```diff
@@ -169,7 +171,7 @@ const blockBasedTrack = types
       const blockWatchDisposer = autorun(() => {
         // create any blocks that we need to create
         const blocksPresent: { [key: string]: boolean } = {}
-        self.blockDefinitions.forEach((block: Instance<BlockStateModel>) => {
+        self.blockDefinitions.forEach((block: any) => {
```

Comment:
any reason why this had to become an any here?

Reply:
nope that's a mistake, gonna remove